app/constants.js

```diff
@@ -0,0 +1 @@
+module.exports.SALTROUNDS = 10;
```

Comment:
we use 
`exports.variable = value` notation
or
`const variable = 5;`
`module.exports = {variable}` notation